src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx

```diff
@@ -68,36 +70,28 @@ describe('lib/viewers/box3d/model3d/Model3DControlsNew', () => {
                 onVrToggle,
             });
 
-            expect(wrapper.find(ResetControl).props()).toMatchObject({
-                onReset,
-            });
-            expect(wrapper.find(AnimationControls).props()).toMatchObject({
-                animationClips: [],
-                currentAnimationClipId: '123',
-                isPlaying: false,
-                onAnimationClipSelect,
-                onPlayPause,
-            });
-            expect(wrapper.find(VrToggleControl).props()).toMatchObject({
-                isVrShown: false,
-                onVrToggle,
-            });
-            expect(wrapper.find(Model3DSettings).props()).toMatchObject({
-                cameraProjection: CameraProjection.PERSPECTIVE,
-                onCameraProjectionChange,
-                onClose: onSettingsClose,
-                onOpen: onSettingsOpen,
-                onRenderModeChange,
-                onRotateOnAxisChange,
-                onShowGridToggle,
-                onShowSkeletonsToggle,
-                onShowWireframesToggle,
-                renderMode: RenderMode.LIT,
-                showGrid: true,
-                showSkeletons: false,
-                showWireframes: false,
-            });
-            expect(wrapper.find(FullscreenToggle).prop('onFullscreenToggle')).toEqual(onFullscreenToggle);
+            await userEvent.click(screen.getByTitle('Reset'));
```

Comment:
Tool creators suggest to initialise userEvent before tests: https://testing-library.com/docs/user-event/intro
Not sure if relevant, just noticed it while checking docs.